test/test_helper.rb

```diff
@@ -17,20 +17,16 @@ module RubyUI
   end
 end
 
-def phlex_context(&)
-  render Phlex::HTML.new, &
-end
-
-module Phlex
-  module Testing
-    module ViewHelper
-      def render(component, &)
-        component.call(view_context:, &)
-      end
+class ComponentTest < Minitest::Test
+  def render(component, &)
+    component.call(view_context:, &)
+  end
 
-      def view_context = nil
-    end
+  def phlex(&)
```

Comment:
Renaming from `phlex_context` to `phlex` just because it's the same name used in phlex codebase: https://github.com/phlex-ruby/phlex/blob/main/fixtures/sgml_helper.rb#L4